### DISCUSSION TITLE
Script working on new ATLAS OS?

### DISCUSSION THREAD

**Discussion by shen-0806** [xeon1989/Proton-X50-APK-Installer#14] [General]
Is that anyone try this script after installed new ATLAS OS? Looking at some comments in X50 fb group, after install new OS, all apps installed before will wiped up from X50 and this script cannot work anymore to install ES file explorer.

**Comment by xeon1989** [comment]
the script most likely will not work with new ATLAS OS.
Do not update if you still want to keep your app.
Unfortunately I do not have solution for anyone to rollback the update yet.

**Reply by shen-0806** [reply]
@xeon1989 noted with thanks! Will not update the OS until have solution to install file explorer and apps again. 
Is that any possible you can make the script work at Atlas OS in future?

**Reply by xeon1989** [reply]
Frankly speaking, unless I have my X50 or hands on ATLAS OS, I have no idea how to make it works in new system. 
I can give it a try but definitely not now, since I am limited by resources I can get.

**Reply by ZiqSyi** [reply]
Thank you for your script before. I hope you will make new script for the new atlas OS in short time. Nothing special on new software. Thanks again xeon1989. Because of you lots of x50 owners can save RM350 & not be bullied by some people who greedy on money.

**Reply by xeon1989** [reply]
You're welcome. 
At least based on what I heard, the new software/OS are different which renders the scripts unusable.
If you really need sideloaded app and, for now I can only advise not to update it before anyone release the new way to do so.
The OS update are not forced as of I aware of.

**Reply by jonnysoe** [reply]
@xeon1989 do you know if there is a way to downgrade the ATLAS OS? And just curious, how did you find those information from the Russian forum?

**Comment by deepblue1004** [comment]
Try the way they install for Persona Premium (also Altas OS)
It may work

**Reply by xeon1989** [reply]
Unfortunately this is not valid yet as of today.
ATLAS for X50 does not come with Spotify yet.
